### PR TITLE
[terra-application][workspace] Square Tabs

### PR DESCRIPTION
**Pull request by dkasper-was-taken**
### Summary
Merge workspace square tabs into staging branch.

### Deployment Link

https://terra-applic-square-tab-sshhib.herokuapp.com

### Testing
Integrated new WDIO pattern.

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: src/workspace/shared/DynamicOverlayContainer.jsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
+import React from 'react';
```
Comment (tbiethman) [inline src/workspace/shared/DynamicOverlayContainer.jsx]: This file is already present in the /src/shared directory. You can double check its contents but you shouldn't need to add it again.

File: src/workspace/shared/StatusLayout.jsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
+import React from 'react';
```
Comment (tbiethman) [inline src/workspace/shared/StatusLayout.jsx]: This is also present in the /src/shared directory

File: src/workspace/shared/StatusLayoutButton.jsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```
Comment (tbiethman) [inline src/workspace/shared/StatusLayoutButton.jsx]: This is also present in the /src/shared directory.

File: src/workspace/shared/usePortalManager.jsx [hunk 4]
```diff
@@ -0,0 +1,87 @@
+import React from 'react';
```
Comment (tbiethman) [inline src/workspace/shared/usePortalManager.jsx]: This is also present in the /src/shared directory.

File: tests/jest/workspace/DynamicOverlayContainer.test.jsx [hunk 5]
```diff
@@ -0,0 +1,186 @@
+import React from 'react';
```
Comment (tbiethman) [inline tests/jest/workspace/DynamicOverlayContainer.test.jsx]: You can remove these net-new test files as well, they should be present in the /tests/jest/shared directory.

File: src/workspace/shared/DynamicOverlayContainer.jsx (hunk 1 above)
Comment (dkasper-was-taken) [reply src/workspace/shared/DynamicOverlayContainer.jsx]: Updated

File: src/workspace/shared/StatusLayout.jsx (hunk 2 above)
Comment (dkasper-was-taken) [reply src/workspace/shared/StatusLayout.jsx]: Updated.

File: src/workspace/shared/StatusLayoutButton.jsx (hunk 3 above)
Comment (dkasper-was-taken) [reply src/workspace/shared/StatusLayoutButton.jsx]: Updated.

File: src/workspace/shared/usePortalManager.jsx (hunk 4 above)
Comment (dkasper-was-taken) [reply src/workspace/shared/usePortalManager.jsx]: Updated.

File: tests/jest/workspace/DynamicOverlayContainer.test.jsx (hunk 5 above)
Comment (dkasper-was-taken) [reply tests/jest/workspace/DynamicOverlayContainer.test.jsx]: Updated.

**Review by tbiethman — APPROVED** [review]

**Review by mjhenkes — APPROVED** [review]

**Comment by mjhenkes** [comment]
Removing the UX Review required label from this PR since this is an incremental PR.

**Pull request merged**